shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -396,17 +396,20 @@ - (void)setupChannels {
 
 - (void)maybeSetupPlatformViewChannels {
   if (_shell && self.shell.IsSetup()) {
+    FlutterPlatformPlugin* platformPlugin = _platformPlugin.get();
     [_platformChannel.get() setMethodCallHandler:^(FlutterMethodCall* call, FlutterResult result) {
-      [_platformPlugin.get() handleMethodCall:call result:result];
+      [platformPlugin handleMethodCall:call result:result];
     }];
 
+    auto platformViewsController = _platformViewsController.get();
```

Comment:
This one should still use the weakptr trick you were using previously.  The reason is that platformViewsController is a C++ object which means it won't get retained when it is captured by the block.  In practice this looks like it would be safe today but I think we should err on the side of caution.

Reply:
@gaaclarke Done.